src/Router/Aura.php

```diff
@@ -24,7 +24,7 @@ class Aura implements RouterInterface
     /**
      * Aura router
      *
-     * @var Aura\Router\Router
+     * @var \Aura\Router\Router
```

Comment:
It would be better to use `AuraRoute` as this class has already been imported with `use Aura\Router\Route as AuraRoute;`